src/thor/bidirectional_astar.cc

```diff
@@ -1023,7 +1053,9 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
          edgelabel_index = edgelabels_forward_[edgelabel_index].predecessor()) {
       const BDEdgeLabel& edgelabel = edgelabels_forward_[edgelabel_index];
 
-      const auto* edge = graphreader.directededge(edgelabel.edgeid(), tile);
+      const auto* edge = get_directed_edge_or_throw("BidirectionalAStar::FormPath failed",
+                                                    graphreader, edgelabel.edgeid(), tile);
```

Comment:
i think  its better without the helper function

```suggestion
      const auto* edge = graphreader.directededge(edgelabel.edgeid(), tile);
      if (edge == nullptr) {
        throw tile_no_longer_available_error_t("BidirectionalAStar::FormPath failed", edgelabel.edgeid());
      }
```

Reply:
Yes, I agree. It was an experiment that I forgot to remove